modules/database/src/ioc/misc/iocInit.c

```diff
@@ -753,5 +766,61 @@ int iocShutdown(void)
 
 static void exitDatabase(void *dummy)
 {
-    iocShutdown();
+    diode_iocShutdown();
 }
+
+
+
+
+/* iocInit */
+static const iocshFuncDef iocInitFuncDef = {"diodeIocInit",0,NULL,
+             "Initializes the various epics components and starts the IOC running.\n"};
+static void iocInitCallFunc(const iocshArgBuf *args)
+{
+    iocshSetError(diode_iocInit());
+}
+
+/* iocBuild */
+static const iocshFuncDef iocBuildFuncDef = {"diodeIocBuild",0,NULL,
+             "First step of the IOC initialization, puts the IOC into a ready-to-run (quiescent) state.\n"
+             "Needs iocRun() to make the IOC live.\n"};
+static void iocBuildCallFunc(const iocshArgBuf *args)
+{
+    iocshSetError(diode_iocBuild());
+}
+
+/* iocRun */
+static const iocshFuncDef iocRunFuncDef = {"diodeIocRun",0,NULL, 
+             "Bring the IOC out of its initial quiescent state to the running state.\n"
+             "See more: iocBuild, iocPause"};
+static void iocRunCallFunc(const iocshArgBuf *args)
+{
+    iocshSetError(diode_iocRun());
+}
+
+/* iocPause */
+static const iocshFuncDef iocPauseFuncDef = {"diodeIocPause",0,NULL,
+             "Brings a running IOC to a quiescent state with all record processing frozen.\n"
+             "See more: iocBuild, iocRub, iocInit"};
```

Comment:
What does rubbing an IOC do, is that like burping a baby? 😆🤓

Reply:
Nah.
Configuration of the Diode works like Aladdin's Magic Lamp: rub the IOC then say what you wish to be replicated.